docs/layouts/_default/baseof.html

```diff
@@ -23,6 +23,25 @@
   {{ hugo.Generator }}
   {{ partial "docs/html-head" . }}
   {{ partial "docs/inject/head" . }}
+    <!-- Matomo -->
+    <script>
+      var _paq = window._paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      /* We explicitly disable cookie tracking to avoid privacy issues */
+      _paq.push(['disableCookies']);
+      /* Measure a visit to flink.apache.org and nightlies.apache.org/flink as the same visit */
+      _paq.push(["setDomains", ["*.flink.apache.org","*.nightlies.apache.org/flink"]]);
```

Comment:
Isn't the documentation always under nightlies?

Reply:
The documentation is, but the project website isn't. By setting this value, we make sure that when a visit is starting at flink.apache.org and then clicks on documentation (and vice versa) is tracked as the same visit and not starting a new one. Else each click would appear as an 'outlink' like the user left our website, which isn't the case.

Reply:
Understood.